k8s/environments/sbox/common-overlay/flux-patch.yaml

```diff
@@ -18,3 +18,97 @@ spec:
   values:
     java:
       image: sdshmctspublic.azurecr.io/toffee/recipe-backend:prod-68644af-20210805161941
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pip-account-management
+  namespace: pip
+  annotations:
+    fluxcd.io/automated: "true"
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/pip/account-management:latest
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pip-business-rules
+  namespace: pip
+  annotations:
+    fluxcd.io/automated: "true"
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/pip/business-rules:latest
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pip-data-management
+  namespace: pip
+  annotations:
+    fluxcd.io/automated: "true"
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/pip/data-management:latest
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pip-publication-services
+  namespace: pip
+  annotations:
+    fluxcd.io/automated: "true"
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/pip/publication-services:latest
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pip-subscription-management
+  namespace: pip
+  annotations:
+    fluxcd.io/automated: "true"
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/pip/subscription-management:latest
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pip-frontend
+  namespace: pip
+  annotations:
+    fluxcd.io/automated: "true"
+spec:
+  values:
+    nodejs:
+      image: sdshmctspublic.azurecr.io/pip/frontend:latest
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pip-channel-management
+  namespace: pip
+  annotations:
+    fluxcd.io/automated: "true"
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/pip/channel-management:latest
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: hmi-case-hq-emulator
```

Comment:
did you mean to add this here?

Reply:
yes.... I believe. It is another project that needs to go up and was implemented in the same method as the pip-frontend